src/main/java/Event.java

```diff
@@ -0,0 +1,38 @@
+public class Event extends Todo {
+    private final String typeRepresentation = "E";
```

Comment:
Perhaps this constant should be capitalised i.e. TYPE_REPRESENTATION. This applies to the typeRepresentation in 
the other classes too.